sql/updates/world/3.3.5/2032_20_44_00_world.sql

```diff
@@ -0,0 +1,7 @@
+DELETE FROM `spell_script_names` WHERE `ScriptName`='spell_gen_black_magic';
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES
+(59630,'spell_gen_black_magic');
+
+DELETE FROM `spell_proc` WHERE `SpellId`=59630;
```

Comment:
This spell proc doesn't exist in db? Maybe update instead of del+insert?

Reply:
I think it's better to delete here, reviewing any existing entry ofc

Reply:
^